api/src/main/java/io/opentelemetry/trace/attributes/SemanticAttributes.java

```diff
@@ -142,18 +142,66 @@
   public static final LongAttributeSetter MESSAGE_UNCOMPRESSED_SIZE =
       LongAttributeSetter.create("message.uncompressed_size");
   /** Database type. For any SQL database, "sql". For others, the lower-case database category. */
```

Comment:
I think this comment is out of date, the list of identifiers has e.g., mysql, postgresql now. How about linking to the list?

https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/database.md#notes-and-well-known-identifiers-for-dbsystem